components/elm/bld/namelist_files/namelist_defaults.xml

```diff
@@ -592,6 +592,7 @@ this mask will have smb calculated over the entire global land surface
 <!-- MALI grids that could be used if the glc_grid logic was working. -->
 <fglcmask hgrid="0.9x1.25" glc_grid="mpas.gis20km">glc/cism/griddata/glcmaskdata_0.9x1.25_Gland5km.nc</fglcmask>  <!-- the CISM one works fine for this mesh -->
 <fglcmask hgrid="0.9x1.25" glc_grid="mpas.ais20km">lnd/clm2/griddata/glcmaskdata_0.9x1.25_60S.nc</fglcmask>
+<fglcmask hgrid="r05"      glc_grid="mpas.gis20km">lnd/clm2/griddata/glcmaskdata_0.5x0.5_everywhere.nc</fglcmask>
 <fglcmask hgrid="r05"      glc_grid="mpas.gis1to10km">lnd/clm2/griddata/glcmaskdata_0.5x0.5_GIS_AIS.nc</fglcmask>
```

Comment:
Do we need to add a glcmask spec for the new `mpas.gis1to10kmR2` mesh too?

Reply:
I'm not sure I understand why there's been no complaint about this. Maybe Jon does? My assumption is that it is somehow working fine using the old mask. Something we should probably understand better.

Reply:
Seeing this again, I remember now that this just needs to extend larger than the GLC mesh so that masked LND cells exist for all GLC mesh cells.  If the existing mask file provides that overlap then we do not need a new mask file (and it would be best to keep the number of mask files to a minimum).  I'll resolve this conversation.